chains/evm/contracts/test/onRamp/OnRamp/OnRamp.generateMessageId.t.sol

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import {Internal} from "../../../libraries/Internal.sol";
+import {OnRamp} from "../../../onRamp/OnRamp.sol";
+import {OnRampSetup} from "./OnRampSetup.t.sol";
+
+contract OnRamp_generateMessageId is OnRampSetup {
+  function _getMetadataHash() internal view returns (bytes32) {
+    return keccak256(
+      abi.encode(Internal.EVM_2_ANY_MESSAGE_HASH, SOURCE_CHAIN_SELECTOR, DEST_CHAIN_SELECTOR, address(s_onRamp))
+    );
+  }
+
+  function test_BasicMessageId() public view {
+    Internal.EVM2AnyRampMessage memory message = _messageToEvent(_generateEmptyMessage(), 1, 1, 100e18, OWNER);
+
+    bytes32 expectedMessageId = message.header.messageId;
+    message.header.messageId = "";
+
+    bytes32 messageId = s_onRamp.generateMessageId(message);
+
+    assertEq(messageId, expectedMessageId);
+  }
+
+  function test_TokenMessageId() public view {
```

Comment:
These tests don't feel like they add too much value? What the actual difference between this one and the one above?